### DISCUSSION TITLE
打开端口转发界面提示服务内部错误: exit status 1

### DISCUSSION THREAD

**Discussion by luoshan5261486** [1Panel-dev/1Panel#5901] [Q&A]
### 1Panel 版本

1.10.12

### 问题描述

debian安装防火墙后打开端口转发界面提示服务内部错误: exit status 1

### 重现步骤

1

**Comment by wanghe-fit2cloud** [comment]
防火墙的端口转发功能依赖服务器上的 iptables 服务，先检查一下 iptables 是否正常运行。
